README.md

```diff
@@ -75,6 +77,7 @@ Awesome! Have a look at the following issues:
 - DevOps ["good first issues"](https://github.com/openedx/wg-devops/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue)
 - Build/Test/Release [bugs](https://github.com/openedx/build-test-release-wg/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
 - Developer Experience ["good first issues"](https://github.com/openedx/wg-developer-experience/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue)
+- Large Instances ["good first issues"](https://github.com/orgs/openedx/projects/42/views/3)
```

Comment:
Note: the label doesn't exist yet, and I can't create it for that repo, but once https://github.com/openedx/terraform-github/issues/51 is completed we should be able to list the labelled tickets there without changing this URL.